app/kvsapp/source/kvsapp.c

```diff
@@ -1112,6 +1174,11 @@ int KvsApp_addFrame(KvsAppHandle handle, uint8_t *pData, size_t uDataLen, size_t
     {
         res = ERRNO_FAIL;
     }
+    else if ((pUserData = (DataFrameUserData_t *)malloc(sizeof(DataFrameUserData_t))) == NULL)
```

Comment:
Should we use `kvsMalloc` here?

Reply:
I intend not to use kvsMalloc inside kvsapp because kvsapp is a sample implementation kvs library. That's why I put kvsMalloc in a private header instead of public header in kvs library.  We can consider move kvsapp to kvs library until kvsapp solid enough.